bcdaworker/worker/alr.go

```diff
@@ -115,6 +115,7 @@ func goWriter(c chan data, w *bufio.Writer,
 func (a *AlrWorker) ProcessAlrJob(
 	ctx context.Context,
 	jobArgs models.JobAlrEnqueueArgs,
+	ndjsonFilename uuid.UUID,
```

Comment:
This is not used, is it planned to be updated at a later time?

Reply:
I think it was a thought, but I don't think so. I'll remove this.